### PR TITLE
Namespaced all variables, Capitalized Tasks, Whitespace fix and removed block in file usage

### LINKED ISSUE #15: Remove trailing whitespace

### PR DESCRIPTION
**Pull request by Scriptkiddi**
Namespaced all variables
Capitalized Tasks
Switched from block in file module to using the ini module for the php.ini
removed trailing whitespaces in all files
Document php_fpm_php_ini_values
fixes #15 #12 #17 #14 #20 #18

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,12 +43,16 @@ php_fpm_pools:
     pm_max_children: 20
     pm_start_servers: 20
     processes_priority: -19
-php_fpm_php_ini_values: 
-  apc.enabled:
-    - 1
-  extension:
-    - php_mysqli.dll
-    - php_ldap.dll
+php_fpm_php_ini_values:
```
Comment (dasJ) [inline README.md:60]: Nice, but have you also thought about modifying the defaults? Feels like #13 is relevant again

File: tasks/main.yml [hunk 2]
```diff
@@ -5,42 +5,39 @@
...
-        {{ key }}={{ value }}
-      {% endfor %} 
-     {% endfor %}
+  ini_file:
+    path: "/etc/php/{{ php_fpm_php_version }}/fpm/php.ini"
+    section: "{{ item.section }}"
+    option: "{{ item.option }}"
+    value: "{{ item.value}}"
+  with_items:
+    php_fpm_php_ini_values
```
Comment (dasJ) [inline tasks/main.yml]: See #13

**Review by dasJ — CHANGES_REQUESTED** [review]

**Comment by dasJ** [comment]
We're getting there

File: README.md (hunk 1 above)
Comment (dasJ) [reply README.md:60]: Also, now that the variable is getting more complex, why not document it in the README?

**Review by dasJ — CHANGES_REQUESTED** [review]

**Comment by Scriptkiddi** [comment]
pls rereview

File: README.md [hunk 3]
```diff
@@ -16,6 +16,19 @@ To see all vars possible for php-fpm see
...
+To pass those vars to ansible you use the list
+`php_fpm_php_ini_values` the syntax for this is as follows
+```
+php_fpm_php_ini_values:
+  - section: SECTION NAME
+    option: OPTION NAME
+    value: VALUE
+```
+| Name                      | Required                 | Description       | 
+|---------------------------|:------------------------:|---------------|
```
Comment (dasJ) [inline README.md]: Oh no, that indentation

**Review by dasJ — APPROVED** [review]

**Review by dasJ — CHANGES_REQUESTED** [review]
Wrong button

**Comment by dasJ** [comment]
Btw, if you are still using my vim config, just go to the table with the cursor and use the `:Tab<Tab>` command

File: README.md (hunk 3 above)
Comment (Scriptkiddi) [reply README.md]: what?

**Comment by Scriptkiddi** [comment]
pls rereview

**Review by dasJ — APPROVED** [review]

**Pull request merged**